tests/lib/cli-engine/config-array/config-array.js

```diff
@@ -693,8 +693,8 @@ describe("ConfigArray", () => {
 
         for (const { filePaths } of [
             { filePaths: [__filename] },
-            { filePaths: [__filename, `${__filename}.ts`] },
-            { filePaths: [__filename, `${__filename}.ts`, path.join(__dirname, "foo.js")] }
+            { filePaths: [__filename, path.join(__dirname, "foo.ts")] },
```

Comment:
@mysticatea Is it a problem if this test is changed?

@gabrieldrs It might be safest to just use `path.basename` and similar functions to figure out the equivalent file path.

Reply:
Updated to `` `${path.resolve(__filename)}.ts` ``, should maintain previous behavior

Reply:
Hmm, if the next character is not a path separator (`/` or `\`), it may be false positive. I think that concatenations to append a new file extension is a valid case because we cannot replace it by `path.join`/`path.resolve`.